### PR TITLE
Upgrade to polkadot v1.1.0

### PR DESCRIPTION
**Pull request by rimbi**
Upgrade dependencies to polkadot v1.1.0

### CODE REVIEW

**Comment by rimbi** [comment]
I will update the branch from `cem-upgrade-to-polkadot-v1.1.0` to `moonbeam-polkadot-v1.1.0` later. Ignore them for now please.

File: Cargo.toml [hunk 1]
```diff
@@ -52,146 +52,146 @@ hex-literal = "0.3.1"
...
-sp-version = { git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.1.0", default-features = false }
-substrate-wasm-builder = { git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.1.0" }
-sp-blockchain = { git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.1.0", default-features = false  }
-sp-consensus = { git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.1.0", default-features = false  }
-sp-storage = { git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.1.0", default-features = false  }
-sp-timestamp = { git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.1.0", default-features = false  }
-sp-trie = { git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.1.0", default-features = false }
+frame-benchmarking = { git = "https://github.com/moonbeam-foundation/polkadot-sdk", branch = "cem-upgrade-to-polkadot-1.1.0", default-features = false }
```
Comment (librelois) [inline Cargo.toml:55]: On our polkadot-sdk fork the branch should be named `moonbeam-polkadot-v1.1.0`

**Review by librelois — CHANGES_REQUESTED** [review]
On our polkadot-sdk fork the branch should be named `moonbeam-polkadot-v1.1.0`

File: Cargo.toml (hunk 1 above)
Comment (rimbi) [reply Cargo.toml]: done

**Review by crystalin — APPROVED** [review]

**Review by girazoki — APPROVED** [review]

**Pull request merged**